README.md

```diff
@@ -32,13 +32,16 @@ To contribute:
 | [Lazada](http://www.lazada.com/work-at-lazada) | Tanjong Pagar | Summer | |
 | [Nutonomy](https://www.nutonomy.com/careers/) | Jurong | All-year-around | Autonomous Driving Intern. Note that you need to be on a MS/PhD track |
 | [OpenGovProducts](https://opengovernmentproducts.recruitee.com/)| Funan| All-year-round| SE & UX interns; Only Singaporean or PR allowed;|
+| [Palantir](https://jobs.lever.co/palantir/?location=Singapore%2C%20Singapore)| Various countries (HQ at Palo Alto) | Unknown | Forward deployed engineer has opportunities for travelling |
+| [PayPal](https://jobsearch.paypal-corp.com/en-US/search?keywords=Engineer%20Intern&location=&facetcountry=sg)| Suntec City | 6 months | |
 | [SEA(Garena)](https://career.seagroup.com/programs?pos=LIP-area)  | One-North | All-year around | |
 | [ShopBack](http://careers.shopback.com/singapore)  | One-North | Summer | No SE intern positions listed |
 | [SEA(Shopee)](https://careers.shopee.sg/) | Kent-Ridge | All-year around | |
 | [Stripe](https://stripe.com/jobs/listing/software-engineering-intern-apac/1806024?gh_src=73vnei) | Outram Park | Summer | |
+| [Titansoft](https://www.titansoft.com/en/career/current-openings?country=singapore) | 90 Eu Tong Sen Street #05-01 | 6 months and summer available | |
```

Comment:
Sorry for spotting this only later - I think for place, we can just leave it to be `Outram Park` to be consistent with all the other entries in the list. On second thought, it doesn't seem like a very useful column given how small SG is.